src/checkbox/stories/Checkbox.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { CheckboxStateReturn, useCheckboxState } from "reakit";
+import { Story, Meta } from "@storybook/react/types-6-0";
+
+import {} from "../index";
+import { Checkbox, CheckboxProps } from "../Checkbox";
+
+export default {
+  title: "Checkbox",
+  component: Checkbox,
+} as Meta;
+
+const Base: Story<CheckboxProps> = args => (
+  <Checkbox {...args}>Checkbox</Checkbox>
+);
+
+export const Default = Base.bind({});
+Default.args = {};
+
+export const DefaultChecked = Base.bind({});
+DefaultChecked.args = { defaultState: true };
+
+export const DefaultIndeterminate = Base.bind({});
+DefaultIndeterminate.args = { defaultState: "indeterminate" };
+
+export const Controlled = () => {
+  const [state, setState] = React.useState<CheckboxStateReturn["state"]>(false);
```

Comment:
If user is gonna use this type often we can export it as a type for ease of use

`type CheckboxStatus = CheckboxStateReturn["state"]`

Reply:
Still not sure, may be take a look. 
This type gives type errors, looking into it.

Reply:
Gives type error? I'm not seeing any issues

Reply:
Fixed it !!